### ISSUE TITLE
Buttons don't have an accessible label name

### ISSUE THREAD

**Issue by Vaibhav-Kumar10** [Shreyaa173/Code-Book#68]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. 
This is missing in the website for search button.

Please assign me this issue as per Extd-GSSoC-2024
